ramls/holdingsrecord.json

```diff
@@ -8,6 +8,10 @@
       "description": "the unique ID of the holdings record; UUID",
       "$ref": "uuid.json"
     },
+    "_version": {
```

Comment:
Hi @hjiebsco, Since the schemas are changed, could you please bump up API versions (`holdings-records-storage`, `item-storage`, `instance-storage`) and also version in the related RAML file?

Reply:
@hjiebsco Please don't merge this pull request (even though it has already been approved) until this has been done.

Reply:
@hjiebsco I agree that this should be a minor API change.

Reply:
Thanks guys. I bumped up the minor version for above three interfaces. Note, `instance-storage.raml` has incorrect version `7.4`, so I changed it to new correct one `7.6`. Just a FYI.

Reply:
Thanks @hjiebsco (alas, that happens often, folks forget to change the interface definition itself)